ts/features/bonus/cgn/components/otp/OtpNotWorking.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from "react";
+import { View } from "native-base";
+import I18n from "../../../../../i18n";
+import {
+  bottomSheetContent,
+  useIOBottomSheetRaw
+} from "../../../../../utils/bottomSheet";
+import ButtonDefaultOpacity from "../../../../../components/ButtonDefaultOpacity";
+import { Label } from "../../../../../components/core/typography/Label";
+import { IOStyles } from "../../../../../components/core/variables/IOStyles";
+import { H4 } from "../../../../../components/core/typography/H4";
+
+type Props = {
+  onPress: () => void;
+};
+
+const OtpNotWorking: React.FunctionComponent<Props> = (props: Props) => (
```

Comment:
```suggestion
const OtpNotWorking = (props: Props) => (
```
Since you don't need to use `props.children`